api_contracts/contracts/downloadCenter/DownloadCenter.md

```diff
@@ -38,9 +38,13 @@ This endpoint returns a list of budget functions with their associated title and
 
 ### Budget Function [GET]
 
++ Parameters
+    + agency: `all` (optional, string)
+    + federal_account: `212` (optional, string)
```

Comment:
These parameters have not yet been implemented in the `/list_budget_functions/` endpoint, and will not be implemented until we migrate to using the Federal Hierarchy data for our top- and sub-tier agency data.